package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this file is not needed because `yarn` should be executed in the `react/` folder, and the relevant `package.json` lives there.

Reply:
Yes, i removed this package.json